### PR TITLE
Code Sniffer: allows php7 return values

### PR DESCRIPTION
**Pull request by bmax**
https://github.com/adobe-community/issues/issues/11364
cc\ @jimdelois @andyroo2000 @mikesherov

### CODE REVIEW

**Comment by andyroo2000** [comment]
👍

File: Behance/Sniffs/Functions/FunctionDeclarationSniff.php [hunk 1]
```diff
@@ -332,4 +356,35 @@ protected function _processFunctionName( PHP_CodeSniffer_File $phpcsFile, $stack
 
   } // _processFunctionName
 
+  /**
+   * @param  PHP_CodeSniffer_File $phpcsFile
```
Comment (jimdelois) [inline Behance/Sniffs/Functions/FunctionDeclarationSniff.php]: Nitpick: FQN.

File: tests/Behance/Sniffs/Functions/FunctionDeclarationSniffTest.inc [hunk 2]
```diff
@@ -81,3 +81,22 @@ function needsAnotherNewLineAfterComment() {
 
   $a++;/**/
 } // needsAnotherNewLineAfterComment
+
+function invalidReturnValue1() : string{}
+function invalidReturnValue2(): string {}
+function invalidReturnValue3() :string {}
+function invalidReturnValue4() :    string {}
+
+function usableReturnValue() : string {
```
Comment (jimdelois) [inline tests/Behance/Sniffs/Functions/FunctionDeclarationSniffTest.inc]: So, I actually never checked with the group if we want `name: returnValue` or `name : returnValue`, but I just assumed the latter.  Looks like an easy fix if people hate it.

**Comment by jimdelois** [comment]
👍

**Pull request merged**